whelk-core/src/main/groovy/whelk/converter/marc/MarcFramePostProcStep.groovy

```diff
@@ -249,6 +263,37 @@ class RestructPropertyValuesAndFlagStep extends MarcFramePostProcStepBase {
         }
     }
 
+    /**
+     * This compares a fuzzy value with stricter values for fuzzy equivalence
+     * when merging a node. We do so by removing defined "fuzz" from the fuzzy
+     * value, and then remove the stricter values as well. If the result is
+     * empty, the fuzzy value is considered equivalent enough.
+     */
+    boolean checkOnlyExpectedValuesInFuzzyProperty(Map target, Map obj,
```

Comment:
This code is not very easy to follow. In particular, how and why the pattern is created and how we actually extract the value we want to check.

I'm (almost) always a fan of splitting things into smaller functions and giving them explanatory names. Would that be possible here?